src/core/catalog.js

```diff
@@ -59,6 +59,21 @@ function fetchDestination(dest) {
   return Array.isArray(dest) ? dest : null;
 }
 
+function fetchRemoteDest(action) {
+  let dest = action.get("D");
+  if (dest) {
+    if (dest instanceof Name) {
+      dest = dest.name;
+    }
+    if (typeof dest === "string") {
+      return stringToPDFString(dest);
+    } else if (Array.isArray(dest)) {
+      return JSON.stringify(dest);
```

Comment:
Don't we need to `stringToPDFString` the dest elements ?

Reply:
I don't *think* so, since it'll be an explicit destination and those don't contain "raw" strings (but only numbers and Names); please see https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/PDF32000_2008.pdf#G11.1696125